README.md

```diff
@@ -0,0 +1,36 @@
+# Hack with docker
+
+To quickly start hacking, hm-admin comes with a docker ready development environment.
+
+It leverages three services:
+
+- `hm-admin`: The interesting part in our case. Hm-admin is running in this container and hm-admin sources are shared with the host
+- `api` and `mysql`: Those containers are responsible for the API to be developed and tested against. The API is made from https://api-platform.com/.
```

Comment:
http://dictionary.cambridge.org/grammar/british-grammar/made-from-made-of-made-out-of-made-with. I think i will stick with from, I like the raw material spirit :deciduous_tree: